meta-elbb-raspberrypi/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,6 @@
+do_install_append() {
+    install -d ${D}${sysconfdir}/systemd/system/sysinit.target.wants
+    lnr ${D}/lib/systemd/system/systemd-time-wait-sync.service ${D}${sysconfdir}/systemd/system/sysinit.target.wants/systemd-time-wait-sync.service
+}
+
+FILES_${PN} += "${sysconfdir}/systemd/system/sysinit.target.wants/systemd-time-wait-sync.service"
```

Comment:
${D} is missing, is it correct?

Reply:
thats correct ... https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#var-FILES
+
https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#var-D

`D` is used at build/install context 

`FILES` is used at packaging context